tests/r/FormatRBearTest.py

```diff
@@ -19,6 +64,37 @@
 
 
 FormatRBearTest = verify_local_bear(FormatRBear,
-                                    valid_files=(good_file,),
-                                    invalid_files=(bad_file,),
+                                    valid_files=(good_file_1,),
+                                    invalid_files=(bad_file_1,),
                                     tempfile_kwargs={"suffix": ".R"})
+
+FormatRBearRarrowTest = verify_local_bear(FormatRBear,
+                                          valid_files=(good_file_2,),
+                                          invalid_files=(bad_file_1,),
+                                          tempfile_kwargs={"suffix": ".R"},
+                                          settings={"r_arrow": "true"})
+
+FormatRBearRcommentTest = verify_local_bear(FormatRBear,
+                                            valid_files=(good_file_3,),
+                                            invalid_files=(good_file_1,),
+                                            tempfile_kwargs={"suffix": ".R"},
+                                            settings={"r_comment": "false"})
+
+FormatRBearRbraceTest = verify_local_bear(FormatRBear,
+                                          valid_files=(good_file_4,),
+                                          invalid_files=(bad_file_1,),
+                                          tempfile_kwargs={"suffix": ".R"},
+                                          settings={"r_brace": "true"})
+
+FormatRBearRBlankTest = verify_local_bear(FormatRBear,
+                                          valid_files=(good_file_1,),
+                                          invalid_files=(bad_file_2,),
+                                          tempfile_kwargs={"suffix": ".R"},
+                                          settings={"r_blank": "false"})
+
+FormatRBearRTabwidthTest = verify_local_bear(FormatRBear,
+                                             valid_files=(good_file_5,),
+                                             invalid_files=(bad_file_1,),
+                                             tempfile_kwargs={"suffix":
+                                                              ".R"},
```

Comment:
and are the tempfile suffixes needed? if you don't need them I would really leave them out, this saves us lines of code we don't need to maintain